Remove `db_cleanup` from startup (#140)
reactive-python/reactpy-django: CHANGELOG.md

## [Unreleased]

### Removed

> -   `django-reactpy` database entries are no longer cleaned during Django application startup. Instead, it will occur on webpage loads, only when database after `REACTPY_RECONNECT_MAX` seconds has elapsed since the last cleaning.

Reviewer: ```suggestion
-   `django-reactpy` database entries are no longer cleaned during Django application startup. Instead, it will occur on webpage loads if `REACTPY_RECONNECT_MAX` seconds has elapsed since the last cleaning.
```
